nebullvm/operations/optimizations/compilers/quantizations/tensor_rt.py

```diff
@@ -23,7 +22,7 @@ def quantize_tensorrt(
 ):
     if quantization_type is QuantizationType.HALF:
         config.set_flag(trt.BuilderFlag.FP16)
-        input_tfms.append(HalfPrecisionTransformation())
+        # input_tfms.append(HalfPrecisionTransformation())
```

Comment:
if not needed anymore just delete the line (Maybe add a short comment about the reason why TensorRT does not need it)